Application/EdFi.Ods.Api/Startup/OdsStartupBase.cs

```diff
@@ -169,9 +192,6 @@ public void ConfigureContainer(ContainerBuilder builder)
 
             void RegisterModulesDynamically()
             {
-                var assemblies = AppDomain.CurrentDomain.GetAssemblies()
-                    .ToList();
-
                 _logger.Debug("Installing modules:");
```

Comment:
Not directly changed here, but this message seems incomplete. Change it to end with a `.`?

Reply:
no this is by design, again to find in the log